data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg

```diff
@@ -679,16 +700,12 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
             type = impact
             range = ranged
             damage = 6
-            number = 1
+            number = 2
             icon = attacks/bolas.png
             [specials]
                 {WEAPON_SPECIAL_USES_ATTACKS 1}
             [/specials]
-            attacks_used = 1
-        [/effect]
-        [effect]
-            apply_to = max_attacks
-            increase = 1
+            attacks_used = 2
```

Comment:
This still has `{WEAPON_SPECIAL_USES_ATTACKS 1}`

Reply:
Meh... but yeah Im prone to making mistakes like that. At least it wasnt anything big. (This time.)